packages/pasteup/typography.ts

```diff
@@ -41,14 +43,42 @@ const fontScaleMapping: any = {
     },
 };
 
-export const fontSize = (typex: string, level: number) =>
-    fontScaleMapping[typex][level].fontSize.toString();
+const fontSizeNumber = (category: Category, level: number): number =>
+    fontScaleMapping[category][level].fontSize;
+
+const lineHeightNumber = (category: Category, level: number): number =>
+    fontScaleMapping[category][level].lineHeight;
+
+const fontSizeCss = (category: Category, level: number): string =>
+    `font-size: ${fontSizeNumber(category, level)}px`;
+
+const lineHeightCss = (category: Category, level: number): string =>
+    `line-height: ${lineHeightNumber(category, level)}px`;
+
+const fontFamily = (category: Category): string => {
+    switch (category) {
+        case 'headline': {
+            return serif.headline;
+            break;
+        }
+        case 'body': {
+            return serif.body;
+            break;
+        }
+        default: {
+            return sans.body;
+            break;
+        }
+    }
+};
 
-export const lineHeight = (typex: string, level: number) =>
-    fontScaleMapping[typex][level].lineHeight.toString();
+const fontfamilyCss = (category: Category): string =>
+    `font-family: ${fontFamily(category)}`;
 
-// (font-size: 14, line-height: 18)
-export const fontScale = (typex: string, level: number): string =>
-    `
-font-size: ${fontScaleMapping[typex][level].fontSize}, 
-line-height: ${fontScaleMapping[typex][level].lineHeight}`;
+// fs('headline', 2) = 'font-size: 16px; line-height: 20px; font-family: GH Guardian Headline, Georgia, serif';
+export const fs = (category: Category, level: number): string =>
```

Comment:
What are your thoughts on using a longer naming scheme, like `fontSize`? It feels like we could get something much more descriptive without it being long and annoying to type/scan.

Reply:
Yeah, we talked about it and @SiAdcock suggested `fs`. I am un-opinionated about it :)

Reply:
`fs` is what frontend uses, which stands for Font Scale. I appreciate that it's non-obvious. Maybe we should use `fontScale`?